lib/core/storage.js

```diff
@@ -179,9 +179,31 @@ export default class Storage {
     if (isUnset(value)) {
       Cookies.remove(_key, _options)
```

Comment:
I haven't looked or understood the logic of the module and call-sites using this code but this code should probably also have server-side branch to remove cookie (regardless if this code is used server-side or not).

Reply:
I think it's being used after fetching the url to redirect back to the original page after a successful login. When the redirect cookie is fetched, it is set to null. So this will not work ssr, unless we add the branch you are talking about. Fixing it now :)